src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -65,7 +71,14 @@ export default function LatestProgressUpdateCardPresentation({
                 >
                     <Tooltip
                         content={tooltipText}
-                        tooltipPosition={{ top: '-2.6rem', right: '-4rem' }}
+                        tooltipPosition={
+                            isDevMode
+                                ? { bottom: '2rem', right: '-2rem' }
+                                : {
+                                      bottom: '1.4rem',
+                                      right: '-5rem',
+                                  }
+                        }
```

Comment:
Are we also changing the previous style?

Reply:
Yes, as previous tooltip position was not accurate

Reply:
When we remove the dev flag, the new style will be applied. So, what's the point of changing the previous style.

Reply:
The backend part of this feature isn’t finished yet, and it might take some time. So, I thought it’d be a good idea to go ahead and fix the tooltip position for now.